addons/website/views/website_visitor_views.xml

```diff
@@ -92,81 +92,59 @@
-                                                t-attf-src="/base/static/img/avatar_grey.png"
-                                                width="54px" height="54px" alt="Visitor"/>
-                                        </div>
-                                        <div class="o_wvisitor_name d-flex flex-grow-1 flex-column my-0 my-lg-2">
-                                            <div class="d-flex align-items-start">
-                                                <b><field name="display_name"/></b>
-                                                <span t-if="record.is_connected.raw_value" class="fa fa-circle text-success ms-2 my-1" aria-label="Online" title="Online"/>
-                                                <span t-else="" class="fa fa-circle text-danger ms-2 my-1" aria-label="Offline" title="Offline"/>
-                                            </div>
-                                            <!-- Double check is necessary for sample view (or error image are shown) -->
-                                            <div t-if="record.country_id.raw_value and record.country_flag.raw_value">
-                                                <img t-att-src="record.country_flag.raw_value"
-                                                    class="o_country_flag" alt="Country"/>
-                                            </div>
-                                        </div>
+                        <div class="oe_kanban_card oe_kanban_global_click">
+                            <div class="o_kanban_card_header d-flex gap-2">
+                                <img t-if="record.partner_image.raw_value"
+                                    t-att-src="kanban_image('res.partner', 'avatar_128', record.partner_id.raw_value)"
+                                    width="54px" height="54px" alt="Visitor"/>
+                                <img t-else=""
+                                    t-attf-src="/base/static/img/avatar_grey.png"
+                                    width="54px" height="54px" alt="Visitor"/>
+                                <div class="o_kanban_card_header_title mb16">
+                                    <div class="d-flex align-items-center gap-2">
+                                        <field name="display_name" class="o_primary"/>
+                                        <span t-if="record.is_connected.raw_value" class="fa fa-circle text-success" aria-label="Online" title="Online"/>
+                                        <span t-else="" class="fa fa-circle text-danger" aria-label="Offline" title="Offline"/>
+                                    </div>
+                                    <!-- Double check is necessary for sample view (or error image are shown) -->
+                                    <div t-if="record.country_id.raw_value and record.country_flag.raw_value">
+                                        <img t-att-src="record.country_flag.raw_value" class="o_country_flag" alt="Country"/>
```

Comment:
@panv-odoo I would add a tooltip with the name of the country. What do you think? Personally I don't know all the flags :) But that's a niptick...